torchao/sparsity/training/README.md

```diff
@@ -42,19 +42,47 @@ swap_semi_sparse_linear_with_linear(model)
 
 ### Benchmarking
 
-If you want to see the expected speedups of applying runtime semi-structured sparsity for training, you can do so by modifying the existing benchmark code in to add your matmul shapes in:
-[benchmarks/benchamrk_semi_sparse.py](https://github.com/pytorch/ao/blob/main/benchmarks/benchmark_semi_sparse.py#L25)
+For ViT-L we see a **6% e2e speedup** on a single NVIDIA A100 across a single training (forwards + backwards) pass with torch.compile enabled and FP16 dtype:
```

Comment:
hmm while this is fine for for this tutorial not sure a top level ao provides 6% training speedups will land as strongly